.vib/spring-cloud-dataflow/goss/spring-cloud-dataflow.yaml

```
@@ -0,0 +1,16 @@
+command:
+  check-spring-cloud-dataflow:
+    # As the version is shown in the output we check it here
+    timeout: 20000
+    exec: timeout --preserve-status 15 java -jar /opt/bitnami/spring-cloud-dataflow/spring-cloud-dataflow.jar
+    exit-status: 143
+    stdout:
+      - {{ .Env.APP_VERSION }}
+      - Started
```

Please add a more detailed stdout check
```suggestion
      - Started DataFlowServerApplication
```